tests/qt_types_standalone/src/lib.rs

```diff
@@ -388,7 +388,7 @@ fn read_qdate(d: &QDate) -> bool {
 }
 
 fn copy_qdate(d: &QDate) -> QDate {
```

Comment:
rename these to `clone_X` as well?

Reply:
Can do they are just test method names...

Reply:
Done